app/controllers/application_controller.rb

```diff
@@ -8,23 +8,27 @@ class ApplicationController < ActionController::Base
   protected
   
   def configure_permitted_parameters
-    columns = %i[first_name last_name phone_number business_name image]
-    devise_parameter_sanitizer.permit(:sign_up, keys: columns)  
+      devise_parameter_sanitizer.permit(:sign_up) { |u|
+        u.permit(:email, :password, :password_confirmation, :employee_profile_attributes => [:first_name, :last_name, :profession_id])
```

Comment:
...`:employee_profile_attributes => [:first_name`... is old syntax. You can use ...`employee_profile_attributes: [:first_name`... instead.